test_cases/brooklyn.json

```diff
@@ -46,7 +46,7 @@
       "type": "dev",
       "in": {
         "text": "brooklyn",
-        "layers": "locality,localadmin",
+        "layers": "borough",
```

Comment:
Two of these four tests are now identical to the other two, and can just be removed instead of being updated.